patterns/2-structured/crossteam-project-valuation.md

```diff
@@ -37,18 +37,19 @@ There is a common process, though, by which you can create a model to calculate
 
 Assemble a small team of subject matter experts in your domain.
 Using that team of experts, estimate 4 things about each consumer of your project output:
-  * How long does it take them to consume your project output?
-  * How long would it otherwise take them to home-roll the value of your project output for themselves?
-  * What percentage of your project output is actually useful for them?
-  * How much time on an ongoing basis (ideally per-use) would they otherwise spend maintaining their home-rolled solution?
+
+* How long does it take them to consume your project output?
+* How long would it otherwise take them to home-roll the value of your project output for themselves?
+* What percentage of your project output is actually useful for them?
+* How much time on an ongoing basis (ideally per-use) would they otherwise spend maintaining their home-rolled solution?
 
 When making these estimations, it's impossible to know with high accuracy _exactly_ how long any activities take.  That's not your goal.
 Rather than exactness, you should strive to _**set a worst-case bound**_ on these estimates.
 The idea is for the group of experts to be able to say to each other, "We don't know exactly how long it would take, but we can all agree it's _at least_ this much."
 Specifically, you should estimate a *maximum* reasonable time to consume your project output and *minimum* reasonable times for consumers to otherwise home-roll, use and maintain their own solutions.
 
-> One note about cost-to home-roll.  The cost to home-roll a solution is NOT necessarily (very unlikely, in fact) the same as the cost of making a shared solution.
-Oftentimes for the same functionality the modularity and quality involved in buliding a cross-team, shared solution makes it a noticeably higher investment than a quick, hardcoded implementation used just once.
+> One note about cost to home-roll.  The cost to home-roll a solution is NOT necessarily (very unlikely, in fact) the same as the cost of making a shared solution.
+> Oftentimes for the same functionality the modularity and quality involved in building a cross-team, shared solution makes it a noticeably higher investment than a quick, hardcoded implementation used just once.
 
 ## Formula
```

Comment:
This pattern contains 3 extra sections that are not part of the regular pattern template format:
- Explanation
- Formula
- Commentary

Some thoughts:
- I wonder if these should be considered sub-sections to "Solution"? (in which case we could move them one heading level down i.e. to `###`. This would likely be the quickest fix to make the format of this pattern more consistent with other.s
- Alternatively we could work the content from the 3 extra sections into other sections?
- **Explanation** could maybe be renamed to **Rationale**?

Reply:
Moving this comment to the main comment thread.